rpm_spec/qubes-utils.spec

```diff
@@ -35,26 +39,38 @@ Common Linux files for Qubes Dom0 and VM
 Summary:    Python package qubesimgconverter
 Requires:   python
 Requires:   pycairo
+%if 0%{rhel} >= 7
```

Comment:
Missing `?`, spec parsing fails - see travis log

Reply:
I'm correcting. Hopefully and helpfully travis is here...